lib/services/user_service.dart

```diff
@@ -846,6 +849,26 @@ class UserService {
     }
   }
 
+  Future<bool> localAuthenticationService(
```

Comment:
Better to pull this into a separate service, instead of adding it to UserService (since it doesn't have anything to do with the User)?

`LocalAuthenticationService.instance.requestAuthentication(context, reason)`?

Reply:
Have named it requestLocalAuthentication as there is already another function called requestAuthentication in the same file.